.devcontainer/Dockerfile

```
@@ -1,8 +1,8 @@
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.205.2/containers/dotnet/.devcontainer/base.Dockerfile
 
 # [Choice] .NET version: 6.0, 5.0, 3.1, 6.0-bullseye, 5.0-bullseye, 3.1-bullseye, 6.0-focal, 5.0-focal, 3.1-focal
-ARG VARIANT="6.0-bullseye-slim"
-FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
+ARG VARIANT="1-8.0-bookworm-slim"
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet:${VARIANT}
```

The image moved to `mcr.microsoft.com/devcontainers/dotnet`
```suggestion
FROM mcr.microsoft.com/devcontainers/dotnet:${VARIANT}
```

https://mcr.microsoft.com/en-us/product/devcontainers/dotnet/tags